paper/paper.md

```diff
@@ -105,10 +105,9 @@ function get_MNIST(;T = Float32)
   return xtrain, ytrain, xtest, ytest
 end
 
-function getdata(; T = Float32) #T for types
+function get_data(; T = Float32)
   ENV["DATADEPS_ALWAYS_ACCEPT"] = "true"
-  # Loading Dataset	
-  xtrain, ytrain, xtest, ytest = get_MNIST(;T)
+  xtrain, ytrain, xtest, ytest = get_MNIST(;T=T)
```

Comment:
For information, `xtrain, ytrain, xtest, ytest = get_MNIST(;T)` also works

Reply:
yes but I think Prof. Orban wants it to be clear, no?

Reply:
You did well, my comment was just for information :)